src/SUMMARY.md

```diff
@@ -40,7 +40,7 @@
   - [Operational](cookbooks/operational/index.md)
     - [Creating a Prototype Data Project on Google Cloud Platform](cookbooks/gcp-projects.md)
     - [Creating Static Dashboards with Protosaur](cookbooks/operational/protosaur.md)
-    - [Scheduling BigQuery Queries in Airflow](cookbooks/bigquery-airflow.md)
+    - [Scheduling Queries](cookbooks/scheduling_queries.md)
```

Comment:
How would you feel about maintaining the existing `bigquery-airflow.md` filename for continuity? Existing links or bookmarks would then still get users to relevant query scheduling docs.

Reply:
Good point, but I think we can handle this via a redirect. Update incoming.